src/features/locators/reducers/addCustomLocator.thunk.ts

```diff
@@ -21,7 +21,7 @@ export const addCustomLocator = createAsyncThunk(
 
     if (getLocatorValidationStatus(message) === ValidationStatus.SUCCESS) {
       try {
-        const foundElementText = JSON.parse(await evaluateXpath(locator.customXpath!, element_id));
+        const { foundElementText } = JSON.parse(await evaluateXpath(locator.customXpath!, element_id));
         let { foundHash } = JSON.parse(await evaluateXpath(locator.customXpath!, element_id));
```

Comment:
could you, please, do a small refactoring: 
```
        const result = JSON.parse(await evaluateXpath(locator.customXpath!));
        const { foundElementText } = result;
        let { foundHash } = result;
```

Reply:
done